File: 11-03-18/c++/01-Mark-the-answer.cpp
#include<bits/stdc++.h>

using namespace std;

int main()
{
    int n,x;
    cin>>n>>x;
    int ar[n];
    for(int i=0;i<n;i++)
        cin>>ar[i];
    int cnt=0;
    bool skip=true;
    
    for(int i=0;i<n;i++)
    {
        if(ar[i]<=x)
            cnt++;
        else if(skip)
            skip=false;
        else
            break;
    }
    cout<<cnt<<endl;
}

File: 11-03-18/python/01-Mark-the-answer.py
n, x = map(int,input().split())
ar = [int(x) for x in input().split()]
cnt = 0
skip = True

for i in range(0,n):
    if(ar[i]<=x):
        cnt+=1
    elif(skip):
        skip=False
    else:
        break
print(cnt)
    
